src/app/components/livegrid.rs

```diff
@@ -51,9 +88,15 @@ pub fn LiveGrid(
       >
         <For
           each=items
-          key=|counter| counter.to_owned()
-          view=move |cx, inscription_id| {
-              view! { cx, <InscriptionItem id=inscription_id/> }
+          key=|item| item.key.clone()
+          view=move |cx, item| {
+              let hash = move || item.hash.get();
+              view! { cx,
+                <li class="relative">
+                  <p class="text-white">{&hash()}</p>
+                  <p class="text-white">{format!("/inscription/{}", & hash())}</p>
+                </li>
+              }
```

Comment:
Current issue: By updating values of `item.hash` no further rendering happened in `For` component or (with current code) it panics with `already mutably borrowed: BorrowError', `